vendor/acutest.h

```diff
@@ -487,7 +487,7 @@ test_print_in_color__(int color, const char* fmt, ...)
         printf("\033[0m");
         return n;
     }
-#elif defined ACUTEST_WIN__ && !defined _GAMING_XBOX_SCARLETT
+#elif defined ACUTEST_WIN__ && (!defined _GAMING_XBOX_SCARLETT || !defined _GAMING_XBOX_XBOXONE)
```

Comment:
```suggestion
#elif defined ACUTEST_WIN__ && !defined _GAMING_XBOX_SCARLETT && !defined _GAMING_XBOX_XBOXONE
```

Reply:
please ignore the previous proposal, here's a simpler variant:

```suggestion
#elif defined ACUTEST_WIN__ && !defined _GAMING_XBOX
```